src/test/scala/nodes/util/MatrixMultiplySuite.scala

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import breeze.linalg.{DenseVector, SparseVector, DenseMatrix}
+import org.scalatest.FunSuite
+import pipelines.Logging
+
+class MatrixMultiplySuite extends FunSuite with Logging {
+  test("Vector Matrix mult bench") {
```

Comment:
Do you want to keep this file or was this just for getting an idea of system limits ?